.github/workflows/maven.yml

```
@@ -27,6 +27,13 @@ jobs:
         with:
           distribution: 'adopt'
           java-version: ${{ matrix.java-version }}
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
```

Based on [these docs](https://github.com/marketplace/actions/setup-java-jdk#caching-packages-dependencies), I think this can just be done with this:

```suggestion
          java-version: ${{ matrix.java-version }}
          cache: 'maven'
```

---

Sorry @ibacher - I didn't see this suggestion when I merged this.  Feel free to add this in / tweak, but things just passed :)